### PR TITLE
 [bitnami/*] Autogenerate json-schema files for all the charts

### PR DESCRIPTION
**Pull request by antgamdia**
### Description of the change

This PR simply performs a bulk update on every chart, generating (or updating) the json-schema file for the `values.yaml` using the [readme-generator-for-helm tool](https://github.com/bitnami-labs/readme-generator-for-helm).

### Benefits

The charts will have an up-to-date values.schema.json file, which, apart from the intrinsic validation benefits, will ease the UX while deploying them using [Kubeapps](https://kubeapps.dev/).

### Possible drawbacks

In the past, we (Kubeapps) ran into issues like https://github.com/bitnami-labs/readme-generator-for-helm/issues/15, but in recent versions, this should no longer be an issue.

### Applicable issues

N/A

### Additional information

TBD

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by rafariossaa — APPROVED** [review]

File: bitnami/postgresql/values.schema.json [hunk 1]
```diff
@@ -1,156 +1,2537 @@
...
+                "schedulerName": {
+                    "type": "string",
+                    "description": "Use an alternate scheduler, e.g. \"stork\".",
+                    "default": ""
+                },
+                "terminationGracePeriodSeconds": {
+                    "type": "string",
+                    "description": "Seconds PostgreSQL primary pod needs to terminate gracefully",
+                    "default": ""
+                },
```
Comment (viceice) [inline bitnami/postgresql/values.schema.json:942]: This is wrong, should allow number.

```
│ Error: values don't meet the specifications of the schema(s) in the following chart(s):
│ postgresql:
│ - primary.terminationGracePeriodSeconds: Invalid type. Expected: string, given: integer

**Comment by viceice** [comment]
I think this should be reverted as there seem to be more broken charts, already found `postgresql` and `matomo`

**Comment by antgamdia** [comment]
Thanks for reporting it. This PR is just updating the schema using the [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm) tool. This is the tool we already use for generating the README file so any issue in the schema will likely be in the documentation as well.

I'll defer the revert decision to my teammates, but IMO, I think finally using the values.yaml annotations in our charts can help us discover bugs, corner cases and improperly documented fields. Once we get rid of them ASAP, the pros of having a proper schema validation plus the easy deployment if using tools like [kubeapps](https://kubeapps.dev/), I personally think are worthwhile.

**Comment by fmulero** [comment]
Several issues have been reported about wrong types and finally we decided to revert these changes.

We have created an internal task to improve the [readme-generator-for-helm tool](https://github.com/bitnami-labs/readme-generator-for-helm) and adapt the `values.yaml` file definition to generate the schemas according the needs of each chart.

Sorry about the inconveniences

**Pull request merged**